packages/coreutils/src/poll.ts

```diff
@@ -158,14 +161,18 @@ export namespace IPoll {
  *
  * @typeparam U - The rejected type of the factory's promises.
  * Defaults to `any`.
+ *
+ * @typeparam V - An optional type to extend the phases supported by a poll.
+ * Defaults to `standby`, which already exists in the `Phase` type.
  */
-export class Poll<T = any, U = any> implements IDisposable, IPoll<T, U> {
+export class Poll<T = any, U = any, V extends string = 'standby'>
```

Comment:
The reliance on type conflation of `'standby'` smells weird.

Reply:
I couldn't figure out how else to give it a default value without side-effects. I'd like the type param to be optional.